### PR TITLE
Minor edits and front matter updates

### PR DESCRIPTION
**Pull request by yeekangc**
Review comments for draft.  For https://github.com/OpenLiberty/blogs/issues/2168.

### CODE REVIEW

File: posts/2022-04-12-guides-updated-mp5-jakartaee9.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ categories: blog
 author_picture: https://avatars3.githubusercontent.com/gkwan-ibm
 author_github: https://github.com/gkwan-ibm
 seo-title: Learn MicroProfile 5.0 and Jakarta EE 9.1 from Open Liberty cloud-hosted guides
-seo-description: Learn how to develop microservices on Open Liberty with MicroProfile 5.0 and Jakarta EE 9.1 from the cloud-hosted guides on the openliberty.io website.
+seo-description: Learn how to develop cloud-native Java applications and microservices on Open Liberty with MicroProfile 5.0 and Jakarta EE 9.1 APIs using cloud-hosted guides on the openliberty.io website.
```
Comment (dmuelle) [inline posts/2022-04-12-guides-updated-mp5-jakartaee9.adoc]:
```suggestion
seo-description: Learn how to develop cloud-native Java applications and microservices on Open Liberty with MicroProfile 5.0 and Jakarta EE 9.1 APIs by using cloud-hosted guides on the openliberty.io website.
```

File: posts/2022-04-12-guides-updated-mp5-jakartaee9.adoc [hunk 2]
```diff
@@ -26,9 +26,9 @@ To learn more about what changed in Jakarta EE 9.1, see the link:https://openlib
 
 == What are the new enhancements in the cloud-hosted guides?
 
-The cloud-hosted guides are enhanced with a new look and updated features. Whether or not you've tried them yet, now is the time to experience how simply you can learn MicroProfile and Jakarta EE features.
+The cloud-hosted guides are enhanced with a new look and updated features. Whether or not you've tried them yet, now is the time to experience how easy you can learn to build applications with MicroProfile and Jakarta EE APIs.
```
Comment (dmuelle) [inline posts/2022-04-12-guides-updated-mp5-jakartaee9.adoc]:
```suggestion
The cloud-hosted guides are enhanced with a new look and updated features. Whether or not you've tried them yet, now is the time to experience how easily you can learn to build applications with MicroProfile and Jakarta EE APIs.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
a couple minor suggestions

**Review by dmuelle — COMMENTED** [review]
just realized if we commit these changes to `draft`  from a patch, they will not be in Gilbert's working branch (`gkwn-ibm-2168`) when he opens a PR to staging

**Comment by dmuelle** [comment]
one option might be to rebase this PR to target `gkwn-ibm-2168` and then PR the changes to draft from that branch. There might be other workarounds though

**Comment by gkwan-ibm** [comment]
Mark this PR as invalid and close it because the branch created from the draft that is no good for production.

**Pull request closed without merging**